kafka-ui-react-app/src/components/Connect/Details/Tasks/__tests__/Tasks.spec.tsx

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
-import { create } from 'react-test-renderer';
-import { mount } from 'enzyme';
-import { containerRendersView, TestRouterWrapper } from 'lib/testHelpers';
+import { TestRouterWrapper } from 'lib/testHelpers';
 import { clusterConnectConnectorTasksPath } from 'lib/paths';
 import TasksContainer from 'components/Connect/Details/Tasks/TasksContainer';
 import Tasks, { TasksProps } from 'components/Connect/Details/Tasks/Tasks';
 import { tasks } from 'redux/reducers/connect/__test__/fixtures';
 import { ThemeProvider } from 'styled-components';
 import theme from 'theme/theme';
+import { render, screen } from '@testing-library/react';
```

Comment:
i think here we have to remove unnecessary , and it is evident from the build error

Reply:
done